llama_index/vector_stores/google/generativeai/genai_extension.py

```diff
@@ -23,7 +22,7 @@
 _DEFAULT_API_ENDPOINT = "autopush-generativelanguage.sandbox.googleapis.com"
```

Comment:
not in this PR, but we should probably update this to prod endpoint

Reply:
Updated to generativelanguage.googleapis.com.